_posts/de/newsletters/2022-10-12-newsletter.md

```diff
@@ -0,0 +1,207 @@
+der Antwort anzusehen.*
+
+[Make AddrFetch connections to fixed seeds][review club 26114]
+ist ein PR von Martin Zumsande, der `AddrFetch` Verbindungen zu
+den [fixen Seeds][fixed seeds] (fix kodierte IP-Adressen) herstellt,
+anstatt sie nur zu `AddrMan` (der Datenbank unserer Peers) hinzuzufügen.
+
+{% include functions/details-list.md
+  q0="Wenn ein neuer Knoten von Grund auf neu startet, muss er sich zunächst mit
+einigen Peers verbinden, von denen er den Initial Block Download (IBD)
+durchführen wird. Unter welchen Umständen verbindet er sich mit den Fixed-
+Seeds?"
+  a0="Nur wenn er nicht in der Lage ist, sich mit Peers zu verbinden, deren
+Adressen von den fix kodierten Bitcoin DNS Seed Nodes bereitgestellt werden.
+Dies tritt am häufigsten auf, wenn der Knoten so konfiguriert ist, dass er
+weder IPv4 noch IPv6 verwendet (zum Beispiel `-onlynet=tor`)."
+  a0link="https://bitcoincore.reviews/26114#l-27"
+
+  q1="Welche beobachtbare Verhaltensänderung führt dieser PR ein?
+Welche Arten von Adressen fügen wir zu `AddrMan` hinzu, und unter welchen
+Umständen?"
+  a1="Anstatt die festen Seeds sofort zu seiner `AddrMan` hinzuzufügen und
+vollständige Verbindungen zu einigen von ihnen herzustellen, stellt der Knoten
+stattdessen `AddrFetch`-Verbindungen zu einigen von ihnen her und fügt die
+_zurückgegebenen Adressen_ zu `AddrMan` hinzu. (`AddrFetch` sind
+Kurzzeitverbindungen, die nur zum Abrufen von Adressen verwendet werden.)
+Der Knoten verbindet sich dann mit einigen der Adressen, die sich jetzt in
+seiner `AddrMan` befinden, um den IBD durchzuführen. Dies führt dazu, dass
+eine geringere Anzahl vollständige Verbindungen zu den Fixed-Seed-Knoten
+hergestellt werden; stattdessen wird versucht mehr Verbindungen aus dem viel
+größeren Set von Knoten, über die uns die Fixed-Seed-Knoten informieren,
+herzustellen. `AddrFetch`-Verbindungen können _jede_ Art von Adressen
+zurückgeben, z.B. `tor`; die Ergebnisse sind nicht auf IPv4 und IPv6 beschränkt."
+  a1link="https://bitcoincore.reviews/26114#l-63"
+
+  q2="Warum würden wir eine `AddrFetch`-Verbindung anstelle einer vollständigen,
+ausgehenden Verbindung zu festen Seeds herstellen wollen? Warum könnte der
+Knotenbetreiber hinter einem festen Seed dies ebenfalls bevorzugen?"
+  a2="Eine `AddrFetch`-Verbindung ermöglicht es unserem Knoten, IBD-Peers aus
+einer viel größeren Anzahl von Peers auszuwählen, was die Gesamtverteilung der
+Netzwerkkonnektivität erhöht. Die Wahrscheinlichkeit, dass Betreiber fester
+Seed-Knoten mehrere gleichzeitige IBD-Peers haben, wäre geringer; was den
+Ressourcenbedarf ihrer Knoten verringert."
+  a2link="https://bitcoincore.reviews/26114#l-77"
+
+  q3="Von den DNS-Seed-Knoten wird erwartet, dass sie ansprechbar sind und
+aktuelle Adressen von Bitcoin-Knoten liefern. Warum hilft das einem
+`-onlynet=tor` Knoten nicht?"
+  a3="Die DNS-Seed-Knoten bieten nur IPv4- und IPv6-Adressen an; sie sind nicht
+in der Lage, einen anderen Adresstyp zu liefern."
```

Comment:
I think that's already a bit weird in the original, since IPv4 and IPv6 are already two different types and there are at least three more networks supported in Bitcoin Core. How about:

```suggestion
in der Lage, andere Adresstypen zu liefern."
```